documentation/docs/pages/operators/nodes/preliminary-steps/vps-setup.mdx

```diff
@@ -35,15 +42,15 @@ Bear in mind that credential signing is primarily CPU-bound, so choose the faste
 
 #### Minimum Requirements
 
-| Hardware | **Minimum Specification**                      |
+| Hardware | **Minimum Specification**                  |
 |----------|--------------------------------------------|
 | CPU      | 8-cores, 2.8GHz base clock speed or higher |
 | RAM      | 16GB DDR4+                                 |
 | Disk     | 500 GiB+ NVMe SSD                          |
 
 #### Recommended Requirements
 
-| Hardware | **Minimum Specification**                       |
+| Hardware | **Minimum Specification**                   |
```

Comment:
ditto

Reply:
These are good for the moment, we are in process of getting new ones yet